src/components/Movie.js

```diff
@@ -0,0 +1,115 @@
+import '../css/Movie.css'
+import React, { Component, Fragment } from 'react'
+import Loading from './Loading'
+import Error from './Error'
+import Trailer from '../components/Trailer'
+import MovieData from '../components/MovieData'
+import endpoints from '../endpoints'
+import star from '../assets/star.png'
+import dollarSign1 from '../assets/dollarSign1.png'
+import dollarSign2 from '../assets/dollarSign2.png'
+
+class Movie extends Component {
+  constructor(props) {
```

Comment:
We could destructure it... but I feel like that's more work than needed on a Sunday night.